mapie/tests/test_classification.py

```diff
@@ -1363,10 +1371,136 @@ def test_get_true_label_position(y_true_proba_place: List[NDArray]):
 
 
 @pytest.mark.parametrize("cv", [5, None])
-def test_error_raps_cv_not_prefit(cv: Union[int, None]):
-    """Test that an error is raised if the method is RAPS
+def test_error_raps_cv_not_prefit(cv: Union[int, None]) -> None:
+    """
+    Test that an error is raised if the method is RAPS
     and cv is different from prefit.
     """
     mapie = MapieClassifier(method="raps", cv=5)
     with pytest.raises(ValueError, match=r".*RAPS method can only.*"):
         mapie.fit(X_toy, y_toy)
+
+
+def test_not_all_label_in_calib() -> None:
+    """
+    Test that the true label cumsumed probabilities
+    have the correct shape.
+    """
+    clf = LogisticRegression()
+    clf.fit(X, y)
+    indices_remove = y != 2
```

Comment:
Not the biggest fan of this method to find indices. A numpy function would be better I believe.

Reply:
done